### DISCUSSION TITLE
Just a bit of feedback wrt v5.6.4 (x86-64)

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1184] [General]
'Huzzah!' would be the right word ~ hard to say how many folks using amiberry on debian/other x86-64 desktop ... I often feel alone =) 

@midwan  ~ You may recall the issues I was encountering with 'Fullscreen' mode on the x86-64, notably worse with nvidia drivers ~ that now seems fixed in v5.6.4 ... afaict this was coincidentally fixed when #1170 was resolved. It goes straight from GUI window to fullscreen at emulation start, and now you can reliably quit the emulation ... which in the nvidia drivers case on this box, resulted in desktop resolution hanging at 800x600 ...works without problems now, and desktop resolution is restored upon quit...

...this also seems to have fixed that other startup quirk on the LFS builds (HD4600 IGP), wherein starting emulation with Fullscreen mode enabled resulted in a 800x600 window at 0,0 that required an Alt-Tab to get fullscreen display of the emulation. The i915 driver never displayed the hanging 800x600 resolution at quit ~ but now at emulation start, it goes straight from GUI to fullscreen cleanly, and also quits the emulation cleanly (albeit a little slowly for some reason), Restart also works as it should ...we're talking win win here ;^)

As I recall it, we'd previously concluded to leave Fullscreen mode dangling as a noted experimental x86-64 quirk, and just use Full-window mode instead ...iirc you were toying with the idea of removing the Fullscreen option, but now that it seems to work as expected, I suppose the debate becomes... in which situations would Fullscreen be preferable to Full-window mode, or offer a measurable gain either way?

I'm not sure... however, there is no longer the case to remove it for in being problematic, as now it works as expected. \o/

I thought it best to report this, pursuant of our previous discussions. In any event, fixed in v5.6.4 on x86-64.

Kudos for your continued improvements with amiberry =)

**Comment by midwan** [comment]
Thanks, I'm happy to see it works as expected now.
The benefits of Full-screen (vs Full-Window) would be mostly seen on low-spec devices, maybe such as the RPI3. A lower resolution would mean less pixels to draw, hopefully giving a few more frames per second to play with.

**Comment by giantclambake** [comment]
Understood - I tend to overlook low-spec hw, my bad ;)
